pkg/volume/util/device_util_linux.go

```diff
@@ -0,0 +1,86 @@
+// +build linux
```

Comment:
No, not supported on OSX. the path /sys/block, for example, does not exist.  We need the _unsupported file.